addons/web/static/src/search/search_model.js

```diff
@@ -513,6 +513,20 @@ export class SearchModel extends EventBus {
         this._notify();
     }
 
+    /**
+     * Remove all 'filter' and 'field' facets.
```

Comment:
Change docstring:
`Removes filter, field and favorite facets but keeps groupBy ones`